src/properties.ts

```diff
@@ -152,6 +154,16 @@ export const properties = {
     default: ".syntest/instrumented",
   },
 
+  // plugins
+  use_plugin: {
+    description: "List of dependencies or paths to plugins to load",
+    items: {
+      type: "string",
+    },
+    default: [],
```

Comment:
Is the plan to define our default plugins here? In that case, we should indicate somewhere that people should copy those over when they want to retain those.

Reply:
I don't think we will have "default plugins" if something should always be in used then it should not be a plugin.

What do you mean by "copy those over"?

Reply:
We could ofcourse have some recommended plugins which would be the default value of this property. But the program should not depend on those plugins. So it's still up to the user to use the plugins or not. I'd say once we have some plugins we should add documentation on what each plugin does and when you should use it.

Reply:
Let's just remove it for now

Reply:
Yeah, we could go for some recommended plugins that are part of this repository. But for now, we can leave it like this.